docs/fieldnorm/README.md

```diff
@@ -1,13 +1,20 @@
+However, the ISO-15189 does not include bioinformatic or software development processes. 
+Fortunately, the IEC-62304 exists, which is a harmonized standard about "Medical device software – Software life cycle processes". 
+The use of this standard remains voluntary, however, it gives good guidance to demonstrate compliance with legal requirements. 
 
-We took the IEC-62304 as a guidance to create a fieldnorm for bioinformatics, to make sure that the software we create is developed and manufactured in accordance with the state of the art taking into account the principles of development life cycle, risk management, including verification and validation, as required by the IVDR (Annex I, point 16). 
+We took the IEC-62304 as a guidance to create a fieldnorm for bioinformatics, to make sure that the software we create is developed and manufactured in accordance with the state-of-the-art taking into account the principles of development life cycle, risk management, including verification and validation, as required by the IVDR (Annex I, point 16). 
 
-The main goal of this document is to give guidance to what should be documented to comply to the ISO-15189 and the IVDR. We tried to make pragmatic solutions, and clarity on what should be documented. 
+The main goal of this document is to give guidance to what should be documented to comply to the ISO-15189 and the IVDR. 
+We tried to make pragmatic solutions, and clarity on what should be documented. 
 
-While writing this fieldnorm, it became clear that the size of bioinformatics teams differs. In particular, when there is only a single bioinformatician working for the medical laboratory, some of the items described in this fieldnorm will be rather impossible. For example, reviewing each others code will be difficult. While the fieldnorm does describe the needs to comply to the IVDR and the ISO-15189, it might be impossible to comply to the fieldnorm due to the lack of personel. Make sure choices about no compliances are reasoned and documented, and revise these choices once the team has expanded. 
+While writing this fieldnorm, it became clear that the size of bioinformatics teams differs. 
+In particular, when there is only a single bioinformatician working for the medical laboratory, some of the items described in this fieldnorm will be rather impossible. For example, reviewing each others code will be difficult. 
+While the fieldnorm does describe the needs to comply to the IVDR and the ISO-15189, it might be impossible to comply to the fieldnorm due to the lack of personnel. 
+Make sure choices about no compliance are reasoned and documented, and revise these choices once the team has expanded. 
 
-This field norm does not include Personnel (6.2 in ISO-15189:2022), as this should be covered by QMS of the Medical Laboratory already. 
+This fieldnorm does not include Personnel (6.2 in ISO-15189:2022), as this should be covered by QMS of the Medical Laboratory already. 
```

Comment:
So I did some research on this correction and it turns out that in English, this `field norm` with a space. It also turns out that this is a concept in physics: https://en.wikipedia.org/wiki/Field_norm

A better word would be "field standard" as I see in the suggestions to translate "veldnorm".

Reply:
I stumbled upon the same problem seeing the space. But in the rest of the document and the referrals (like our nice abstract) we always spoke about fieldnorn. Therefor I just removed the space to keep things at least the same. But I completely agree, field standard would be way better.

Reply:
I agree that consistency is better until we agree to use a different term.